index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="main.css">
+    <script src="https://kit.fontawesome.com/e98a0f000a.js" crossorigin="anonymous"></script>
+    <title>About Page</title>
+</head>
+<body>
+        <header class="header">
+            <nav class="nav">
+                <div class="logodiv">
+                    <ul class="headercontent">
+                        <li> 
+                            <i class="language nav1-link fab fa-facebook-f"></i>
+                        </li>
+                        <li> 
+                            <i class="language nav1-link fab fa-twitter"></i>
+                        </li>
+                        <li><a class="lang">English</a></li>
+                        <li><a class="lang">My Page</a></li>
+                        <li><a class="lang">Logout</a></li>
+                    </ul>
+                </div>
+                <section class="sectionlogo pop-up hidden" id="about">
+                 <div class="logo">
+                        <a href="index.html"> <img class="imglogo" src="img/Era.png" alt="era logo"></a>
+                    </div> 
+                    <!-- <div class="hamburger">
+                        <img src="img/icon-X.svg" class="mobileclose" id="closeid" alt=""> 
+                    </div> -->
+                    <div class="items"> 
+                        <ul class="listitems" id="up-menu">
+                            <li class="close">&times;</li>
+                            <li> <a href="about.html" class="about">About</a></li>
+                            <li><a class="listitems1" href="#">Program</a></li>
+                            <li><a class="listitems1" href="#">Join</a></li>
+                            <li><a class="listitems1" href="#">Sponsor</a></li>
+                            <li><a class="listitems1" href="#">News</a></li>
+                            <li><a class="listitems2" href="#">Campaign</a></li>
+                        </ul>
```

Comment:
- [ ] Please make sure to follow the given design and provide a proper mobile menu styling to your website.
![Screenshot (3)](https://user-images.githubusercontent.com/47625407/164485338-bbd1dd40-2033-45a3-9c87-a64690552098.png)